File: GenerateRandomPointinaCircle2.py
class Solution:

    def __init__(self, radius: float, x_center: float, y_center: float):
        self.radius = radius
        self.x_center = x_center
        self.y_center = y_center

    def randPoint(self):
        import random, math
        theta = random.uniform(0, 2 * math.pi)
        l = math.sqrt(random.uniform(0, self.radius ** 2))
        return [l * math.cos(theta) + self.x_center, l * math.sin(theta) + self.y_center]


if __name__ == '__main__':
    radius = float(input())
    x_center = float(input())
    y_center = float(input())
    solution = Solution(radius, x_center, y_center)
    print(solution.randPoint())

File: GenerateRandomPointinaCircle2.cpp
#include<iostream>
#include<vector>
#include<math.h>

using namespace std;

class Solution {
private:
	double radius, x_center, y_center;
public:
    Solution(double radius, double x_center, double y_center) {
        this->radius = radius;
        this->x_center = x_center;
        this->y_center = y_center;
    }
    
    vector<double> randPoint() {
        double theta = 2 * M_PI * ((double)rand() / RAND_MAX);
        double len = sqrt((double)rand() / RAND_MAX) * radius;
        return {x_center + len * cos(theta), y_center + len * sin(theta)};
    }
};

int main() {
	
	double radius, x_center, y_center;
	cin>>radius>>x_center>>y_center;
	
	Solution *solution = new Solution(radius, x_center, y_center);
	
	for(int i = 0; i < 5; i++) {
		vector<double> point = solution->randPoint();
	
		cout<<point[0]<<" "<<point[1];
		cout<<endl;
	}
	
	return 0;
}
